### ISSUE TITLE
Dragon's Breath

### ISSUE THREAD

**Issue by nstevens26** [Cards-of-Chaos/Game-Design#36]
You summon the might of a dragon to your throat allowing you to expel their breath weapon.

50/50:

Fire Dragon: You breath out fire in a 4 foot cone in front of you. All creatures in this cone take 16 fire damage and must make a spell saving throw. On a failed save, the creature is burned.

Ice Dragon: You breath out ice in a 4 foot cone in front of you. All creatures in this cone take 13 ice damage and must make a spell saving throw. On a failed save, the creature is immobilized.

Electric Dragon: You breath out electricity in a 4 foot cone in front of you. All creatures in this cone take 15 electric damage and must make a spell saving throw. On a failed save, the creature is Static.

Wind Dragon: You breath out forceful wind in a 4 foot cone in front of you. All creatures in this cone take 16 wind damage and must make a spell saving throw. On a failed save, the creature is moved 3 spaces backwards and is deafened.

**Comment by nstevens26** [comment]
What class should this spell be moved to for now?

**Comment by nstevens26** [comment]
Moved to class Mage

**Comment by D-D-2** [comment]
I agree with this being put into mage. But it's also making me think about the new shaman class as well. How do we want to define Shaman spells vs Mage/summoner spells?

**Comment by nstevens26** [comment]
This spell will need range of the breath weapons added. I could see this spell falling under both Mage and Shaman.
